Gopkg.lock

```diff
@@ -405,6 +413,14 @@
   revision = "e9657d882bb81064595ca3b56cbe2546bbabf7b1"
   version = "v1.4.0"
 
+[[projects]]
+  digest = "1:fe9eb931d7b59027c4a3467f7edc16cc8552dac5328039bec05045143c18e1ce"
+  name = "gopkg.in/h2non/gock.v1"
```

Comment:
Please add this package to the Gopkg.toml, run `dep ensure` and commit the vendor dir to git.

Reply:
Gopkg.toml and vendor updated, thanks.